### ISSUE TITLE
Imported subdomains are not generated by PlantUML generator

### ISSUE THREAD

**Issue by tk** [ContextMapper/context-mapper-dsl#321]
**Describe the bug**
A model distributed over several files, does not generate subdomain PlantUML diagrams of the imported domains.

**To Reproduce**
1. Create Domain with Subdomain in other cml file
2. Generate PlantUML diagrams for "import-to-other-domain.cml"
3. See list of generated PlantUML diagrams:
    - src-gen/import-to-other-domain_BC_context1.puml
    - src-gen/import-to-other-domain_BC_context2.puml
    - src-gen/import-to-other-domain_ContextMap.puml
    - src-gen/import-to-other-domain_SD_DirectSubdomain.puml

**Expected behavior**
The file "src-gen/import-to-other-domain_SD_OtherSubdomain.puml" is also created.

**Input files to reproduce**
File: import-to-other-domain.cml
```
import "./other-domain.cml"

Domain DirectDomain {
	
	Subdomain DirectSubdomain {
		
		Entity DirectEntity
	}
}

ContextMap testMap {

}

BoundedContext context1 implements DirectDomain
BoundedContext context2 implements OtherSubdomain
```
File: OtherDomain.cml
```
Domain OtherDomain {
	
	Subdomain OtherSubdomain {
		
		Entity OtherEntity
	}
}
```
**IDE and Plugin (please complete the following information):**
 - Eclipse version 2022-03 (4.23.0)
 - ContextMapper plugin version 6.6.1

**Additional context**
I don't know if this behavior is intended or not. But from my understanding of the [Imports](https://contextmapper.org/docs/imports/) language feature, I would expect the generation of all model elements, as if they were in the same file.

**Comment by stefan-ka** [comment]
Hi @tk 
Thanks for reporting the issue and sorry we haven't responded so far.
I will try to have a look at this one for the next release.
Best regards,
Stefan

**Comment by stefan-ka** [comment]
Will be fixed with upcoming release v6.11.0.

**Comment by dotnich-io** [comment]
I'm not sure about the internal architecture of this dsl, but I came here to report the same surprising behavior from the VS Code extension and would be delighted if the v6.11.0 release fixed the behavior there also :)

**Comment by stefan-ka** [comment]
Fixed in v6.11.0 which was just released (Eclipse + VS Code).
